### ISSUE TITLE
Crossroads - magic and tech mod

### ISSUE THREAD

**Issue by Spontaneo** [EnigmaticaModpacks/Enigmatica6#3774]
### URL

https://www.curseforge.com/minecraft/mc-mods/crossroads-mc

### Why would you like the mod added?

A neat mod that is rather expansive in what it can do.  It is similar to create in that it relies on creativity to be more effective rather than a straight forward path like mechanism.  It has 2 main branches, Alchemy which is focused on weird items and resource generation while Magitech is focused on weird mechanics such as speeding up or slowing down time in a small area not to be mixed up with increasing tic rate.  Has a bunch of neat and weird mechanics that would play nice with other mods.  There is a spotlight video on the page that can go into better details than i could.

**Comment by NillerMedDild** [comment]
Thanks for the suggestion, I'll think about it. But, chances are slim I'd say - Adding both this and Essentials would be quite a bit of extra content, and the pack is already huuuge.

**Comment by Ridanisaurus** [comment]
To be honest, I tried that mod a while ago and It was underwhelming. The art leaves quite a bit to be desired, and most of its mechanics are covered by other mods. If you don't want to do the whole 1-magic-block-machine then Create is the answer. If you want to use Magic to process stuff, then Blood Magic has that covered. 
It felt like playing with a bootleg version of Create the last time I played with it, but do note that was a while ago, and things might have changed since, but at a quick glance on their Curseforge Page, things seem to be pretty much the same. 
It does have protentional, but it's not there yet.
Also, if we end up adding that to the modpack, we either need to find a way to disable all its sounds (dunno if there is a config for that) or ship them muted through the muffler. The sound effects used by the mod are choppy, loud, and simply unpleasant. Felt like wearing a bad headphone and it's crackling randomly every now and then.
